### ISSUE TITLE
Add git commit hooks for prettier

### ISSUE THREAD

**Issue by EdrianI** [openeew/openeew-dashboard#52]

**Comment by EdrianI** [comment]
It was mentioned on 1/29 during the Dashboard meeting that we making use of the Husky package. 

As of today, the Husky package is only installed as part of the client's setup. It is also not working. The Husky documentation recommends that we use the root directory because Husky makes use of the .git directory.  https://typicode.github.io/husky/#/?id=monorepo

Furthermore, it also recommends that we use lerna to deal with multi-package repos. https://github.com/lerna/lerna

**Comment by EdrianI** [comment]
It was mentioned on 1/29 during the Dashboard meeting that we making use of the Husky package. 

As of today, the Husky package is only part of the client's package.json. As-is it is not working, I tested this by doing the following:

1. npm run setup from /web
2. changing line 46 of the /client/package.json to "echo derp && exit 1" 
3. and trying to commit.

This basic test should make the commit fail. I'm opening a Pull Request to move this devDependency up to web/package.json.

FYI:
Husky 5.0+ is now out and has several breaking changes. The way we are using husky by adding a husky section to the package.json is no longer an option in >5.0. Instead Husky creates a .husky directory that contains the git hooks and some scripts. 

If we moved to 5.0, the Husky documentation recommends that we use the root directory because Husky makes use of the .git directory.  https://typicode.github.io/husky/#/?id=monorepo

Furthermore, it also recommends that we use lerna to deal with multi-package repos. https://github.com/lerna/lerna
